### DISCUSSION TITLE
MapsForgeTileSource.createInstance crashes since version 0.16

### DISCUSSION THREAD

**Discussion by johan974** [mapsforge/mapsforge#1531] [Q&A] [answered]
When using mapsforge for over 10 years (icm Osmdroid) I notice that the following statement crashes starting from 0.16. 

MapsForgeTileSource.createInstance(WaypointActivity.thisActivity.getApplication());

I start via a layout file the Mapsforge map facility. I cannot get any stacktrace but the class in the layout file (HibridMap) failing. 

In that class I use the following statements: 

```
MapsForgeTileSource.createInstance(WaypointActivity.thisActivity.getApplication());
fromFiles = MapsForgeTileSource.createFromFiles(maps, null, "rendertheme-v4");
```

When upgrading to 0.16, the first statements fails. Why? Any change on that one? 
When skippling the createInstance, I see an 'empty' map file. It won't crash anymoren. 

My dependencies section contains: 
```
    implementation 'com.github.MKergall:osmbonuspack:6.9.0'
    implementation 'org.osmdroid:osmdroid-android:6.1.14'
    implementation 'org.osmdroid:osmdroid-mapsforge:6.1.14'
    implementation 'org.mapsforge:mapsforge-map-android:0.15.0'
    implementation 'org.mapsforge:mapsforge-map:0.15.0'
```

It would be great if anyone can help!

**Comment by devemux86** [comment, marked as answer]
`MapsForgeTileSource` is not a Mapsforge class.

Better ask questions about Osmdroid in their repository and check their samples.
